internal/services/monitor/monitor_data_collection_rule_resource_test.go

```diff
@@ -734,5 +734,5 @@ resource "azurerm_resource_group" "test" {
   location = "%[2]s"
 }
 
-`, data.RandomInteger, data.Locations.Primary)
+`, data.RandomInteger, "westeurope")
```

Comment:
rather than doing this, can we set `data.Locations.Primary` to `westeurope` in the test method?